src/main/java/prefs/UserConfig.java

```diff
@@ -0,0 +1,19 @@
+package prefs;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Test class to see if the file handler works with other config subclasses in the new framework
+ */
+public class UserConfig implements Config {
```

Comment:
Are the keyboard shortcuts something we can move in here? Then we don't have to get rid of this, only to add it again later.